### DISCUSSION TITLE
Notification positioning

### DISCUSSION THREAD

**Discussion by lorenlang** [php-flasher/php-flasher#25] [Q&A]
The site I'm working on has a fixed header and the notification is positioned behind it.  I've tried changing the z-index in the template but it didn't make any difference.  Is there some way to force the notification to always be in front of the fixed header div?

**Comment by yoeunes** [comment]
Hello @lorenlang, do you have any repo where you can reproduce the problem ?
for now i can only think of z-index, otherwise can you give the notification a margin top so you can show it under the fixed navbar ?

**Reply by lorenlang** [reply]
Margin top didn't work either. The app is hosted on our internal git server and only works with access to our network.  But I'll put together a generic proof of concept using the base templates that I have and put that in a public repo here on Github.  I'll let you know when that's done.

**Reply by lorenlang** [reply]
This should work as a basic laravel project. https://github.com/lorenlang/flasher-example
Initial page load will be an empty block with the fixed sidebar and header.  Click the Home link in the sidebar and it will redirect back to the same page with the success message.

**Reply by yoeunes** [reply]
Can you please add the z-index to flasher-container, and test if it works for you ?

```css
<style>
   #flasher-container-top-right {
    z-index: 10;
  }
</style>
```

take a look at this PR : https://github.com/lorenlang/flasher-example/pull/1

**Reply by lorenlang** [reply]
I added that rule to my main css file and it worked great.  Thanks!

**Reply by yoeunes** [reply]
Great, I'll try to document this next time, thank you
